org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java

```diff
@@ -90,8 +90,8 @@ InitializeResult initialize(InitializeParams param){
 		capabilities.setWorkspaceSymbolProvider(Boolean.TRUE);
 		capabilities.setReferencesProvider(Boolean.TRUE);
 		capabilities.setDocumentHighlightProvider(Boolean.TRUE);
-		capabilities.setDocumentFormattingProvider(Boolean.TRUE);
-		capabilities.setDocumentRangeFormattingProvider(Boolean.TRUE);
+		//		capabilities.setDocumentFormattingProvider(Boolean.TRUE);
```

Comment:
the capabilities should be set during init, if the client doesn't support dynamic capabilities